.werft/wipe-devstaging.ts

```diff
@@ -1,23 +1,73 @@
-import { werft } from './util/shell';
+import { werft, exec } from './util/shell';
 import { wipePreviewEnvironment, listAllPreviewNamespaces } from './util/kubectl';
+import  * as fs from 'fs';
+import { sleep } from './util/util';
 
 
-async function wipeDevstaging() {
+async function wipeDevstaging(pathToKubeConfig: string) {
```

Comment:
I won't find time for a full review today, but: please let's try to find another way to switch between kubectl contexts. This really decreases the signal to noise ratio a lot.

I have not coherent view on this or what might be better, but will try to come up with sth tomorrow.

Reply:
One way I can think of is to merge all the kubeconfigs we have into the default one and then use context switch. That would involve downloading a kubeconfig -> updating the context name in that and then adding it to the default kubeconfig.

Since pathToKubeConfig is how I have done for all other places I would let this PR merge and raise a separate PR to use a new approach for all the files.

Reply:
> Since pathToKubeConfig is how I have done for all other places I would let this PR merge and raise a separate PR to use a new approach for all the files.

Ok, don't want to block this PR as it fixes sweeper. But let's not forget this.

> One way I can think of is to merge all the kubeconfigs we have into the default one and then use context switch.

This was the way we used to do it in the past albeit it was simpler back then because clusters were static. But we can also leave the separate kubectl configs, but pass a more generic `env` around which `exec` supports (we already have that in some places).

Reply:
>This was the way we used to do it in the past albeit it was simpler back then because clusters were static. But we can also leave the separate kubectl configs, but pass a more generic env around which exec supports (we already have that in some places).

This seems like a much better solution! I will try this out and raise a PR